locales/en.yml

```
@@ -28,28 +28,39 @@ myprofile:
   posts:
     add: Write a post
   title: My Profile
-chat:
+  chat:
```

this change puts chat under `myprofile.chat`
```suggestion
chat:
```